pages/linux/brew.md

```diff
@@ -10,13 +10,13 @@
 
 `brew install {{formula}}`
 
-- List installed formulae [with matching name]:
+- List installed formulae, argument `formula` is optional:
 
-`brew list {{[text]}}`
+`brew list {{formula}}`
```

Comment:
I think I was trying to cover every detail in this command, which more or less went against guideline 1&3: 
> it's OK if the page doesn't cover everything
> The goal is to allow people to understand the syntax of the commands

Now, I prefer we just keep `brew list`, and nothing more about this command.